pkg/controller/ps/tls_test.go

```diff
@@ -15,8 +18,132 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
+
+	apiv1alpha1 "github.com/percona/percona-server-mysql-operator/api/v1alpha1"
 )
 
+var _ = Describe("TLS secrets without cert-manager", Ordered, func() {
+	ctx := context.Background()
```

Comment:
Just create a namespace for the test like it is done in `Finalizer delete-ssl` test for example. If we use the default namespace for different tests I'm sure we will have strange stuff happening.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/297/commits/8726b1189e1c0af947fb35f64107e1a7cff72c68